readthedocs/projects/models.py

```diff
@@ -1570,6 +1570,30 @@ def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
         super().save(*args, **kwargs)
 
 
+class HTTPHeader(TimeStampedModel, models.Model):
+    """
+    Define a HTTP header for a user Domain.
+
+    All the HTTPHeader(s) associated with the domain are added in the response
+    from El Proxito.
+
+    NOTE: the available headers are hardcoded in the NGINX configuration for
+    now (see ``dockerfile/nginx/proxito.conf``) until we figure it out a way to
+    expose them all without hardcoding them.
+    """
+
+    domain = models.ForeignKey(
+        Domain,
+        related_name='http_headers',
+        on_delete=models.CASCADE,
+    )
+    name = models.CharField(max_length=128)
+    value = models.CharField(max_length=256)
+
+    def __str__(self):
+        return self.name
```

Comment:
```suggestion
        return f"HttpHeader: {self.name} on {self.domain.domain}"
```

Suggestion (nonblocking): I think we probably want a bit more detail here, so we know what the object is when printed.